policy/modules/contrib/frr.fc

```diff
@@ -0,0 +1,25 @@
+/usr/lib/frr(/.*)?              gen_context(system_u:object_r:frr_exec_t,s0)
+
+/usr/lib/systemd/system/frr.*   gen_context(system_u:object_r:frr_unit_file_t,s0)
+
+/etc/frr(/.*)?                  gen_context(system_u:object_r:frr_conf_t,s0)
+
+/var/log/frr(/.*)?              gen_context(system_u:object_r:frr_log_t,s0)
+/var/tmp/frr(/.*)?             gen_context(system_u:object_r:frr_tmp_t,s0)
+
+/var/lock/subsys/bfdd           gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/bgpd           gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/eigrpd         gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/fabricd        gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/isisd          gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/nhrpd          gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/ospf6d         gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/ospfd          gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/pbrd           gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/pimd           gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/ripd           gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/ripngd         gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/staticd        gen_context(system_u:object_r:frr_lock_t,s0)
+/var/lock/subsys/zebra          gen_context(system_u:object_r:frr_lock_t,s0)
```

Comment:
Are all of the subsys locks plain files?

Reply:
Yes, It was tested.

Reply:
For plain files the `--` spec can be used.